### PR TITLE
Add binding for X509_load_crl_file

### PR DESCRIPTION
**Pull request by steffen-eiden**
Signed-off-by: Steffen Eiden <seiden@linux.ibm.com>

### CODE REVIEW

**Comment by steffen-eiden** [comment]
I can add at test if #1123 is merged, as before there is no crl in `openssl/test`

Or did I miss one?

File: openssl/src/x509/store.rs [hunk 1]
```diff
@@ -211,6 +211,24 @@ impl X509LookupRef<File> {
...
         }
     }
+
+    #[corresponds(X509_load_crl_file)]
+    /// Specifies a file from which certificate revocation lists will be loaded
+    pub fn load_crl_file<P: AsRef<Path>>(
+        &mut self,
+        file: P,
+        file_type: SslFiletype,
+    ) -> Result<(), ErrorStack> {
```
Comment (sfackler) [inline openssl/src/x509/store.rs:221]: Probably makes sense to propagate the return value since it indicates the number of loaded CRL entries.

**Comment by sfackler** [comment]
Yeah I don't think we have an existing CRL for tests.

File: openssl/src/x509/store.rs (hunk 1 above)
Comment (steffen-eiden) [reply openssl/src/x509/store.rs:221]: I second that. However, the corresponding cert function does also not return that value, so I kept it returning `void`

I can change `load_cert_file` to returning ` Result<i32, ErrorStack` as well If that is OK for you.

Comment (sfackler) [reply openssl/src/x509/store.rs:221]: That'd be a breaking change so I wouldn't want to do it now. Can you just add a FIXME to change that signature to match load_crl_file?

Comment (steffen-eiden) [reply openssl/src/x509/store.rs:221]: OK I'll do that

**Pull request merged**